main_test.go

```diff
@@ -84,11 +81,13 @@ func init() {
 		cmd.Stderr = os.Stderr
 		err := cmd.Run()
 		if err != nil {
+			log.Printf("error while checking testbackend %q: %s", be.name, err.Error())
```

Comment:
could you append the displaying of the error in the `disabling ...` printout instead?

Reply:
Done.